Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```diff
@@ -0,0 +1,62 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.com)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Client id for grant type
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client secret for grant type
+  hidden: false
+  name: client_secret
+  required: true
+  type: 0
+description: Manage credentials for applications, databases, CI/CD tools, and services
+  without causing friction in the development process.
+display: ThycoticDSV
+name: ThycoticDSV
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Name secret for operation Get
```

Comment:
Let's fix the descriptions, make them more clear and correct from a grammar perspective.
Applies to this description, and to all other descriptions too.